structuredheadings/plugin.js

```diff
@@ -307,7 +308,8 @@
               editor.elementPath().block.addClass(editor.config.autonumberBaseClass);
               setStyle(editor, editor.elementPath().block, editor.config.autonumberCurrentStyle);
             }
-
+          } else if (editor.elementPath().block.is({li: 1})) {
```

Comment:
It's a bit confusing how this works, but it might be better to use blockLimit to determine if we're in a list.  In this case I think it might only work if there's no nested block (table or paragraph inside a list item)

But that might be intended.  I'm not sure.